packages/amplify-e2e-core/src/utils/index.ts

```diff
@@ -25,7 +25,15 @@ export * from './sleep';
 export * from './transformConfig';
 export * from './admin-ui';
 export * from './hooks';
-export * from './transform-current-project-to-git-pulled-project';
+export * from './git-operations';
+
+/**
+ * Whether the current environment is CircleCI or not
+ */
+export const isCI = (): boolean => !!(process.env.CI && process.env.CIRCLECI);
```

Comment:
`!!` ?

Reply:
lol I copy pasted this and didn't even realize that's totally not needed